langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockAnthropicStreamingChatModel.java

```diff
@@ -30,4 +41,36 @@ public enum Types {
             this.value = modelID;
         }
     }
+
+    @Override
+    public void generate(final UserMessage userMessage, final StreamingResponseHandler<AiMessage> handler) {
+        List<ChatMessage> sanitizedMessages = sanitizeMessages(List.of(userMessage));
+        super.generate(sanitizedMessages, handler);
+    }
+
+    @Override
+    public void generate(final List<ChatMessage> messages, final List<ToolSpecification> toolSpecifications, final StreamingResponseHandler<AiMessage> handler) {
+        List<ChatMessage> sanitizedMessages = sanitizeMessages(messages);
+        super.generate(sanitizedMessages, toolSpecifications, handler);
```

Comment:
this will throw an exception anyway, please remove this overriden method

Reply:
removed